### PR TITLE
Integrations: Add Prometheus, Grafana and Argo 2986

### LINKED ISSUE #2986: [Meshery] Integrations: Add Prometheus, Grafana and Argo
**Current Behavior**

The Meshery has three more upcoming integration which has not been added to the [Integration Collection](https://layer5.io/service-mesh-management/meshery/integrations) page. 

**Desired Behavior**

- Prometheus, Grafana and Argo should be on the [Integration Collection](https://layer5.io/service-mesh-management/meshery/integrations) Page.
- These images should be an SVG file.

---

<h4>Contributor Resources</h4>

The layer5.io website uses Gatsby, React, and GitHub Pages. Site content is found under the [`master` branch](https://github.com/layer5io/layer5/tree/master).
- See [contributing instructions](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md)
- See Layer5 site designs in this [Figma project](https://www.figma.com/file/5ZwEkSJwUPitURD59YHMEN/Layer5-Designs). Fill-in the [Layer5 Community Member Form](https://layer5.io/newcomer) and join the [Layer5 Community Slack](http://slack.layer5.io) for access.

### PR DESCRIPTION
**Pull request by abhiyant-10**
**Description**

This PR fixes #2986 

**Notes for Reviewers**

**[Signed commits](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [ ] Yes, I signed my commits.

### CODE REVIEW

**Comment by sahilsuman933** [comment]
LGTM! But wait for the workflow approval.

**Comment by l5io** [comment]
🚀 Preview for commit 0040e76f020b39dcfca0149271f337f1daba39b7 at: https://62ef9a0f8e24d678e7d926c1--layer5.netlify.app

**Review by sahilsuman933 — APPROVED** [review]

**Comment by sahilsuman933** [comment]
@Nikhil-Ladha, can you take a look at this PR and merge it?

File: src/assets/data/integrations/IntegrationList.js
Comment (Nikhil-Ladha) [inline src/assets/data/integrations/IntegrationList.js:205]: The status is defined in multiple ways somewhere it is `inProgress` while somewhere it is `InProgress`. Please make it consistent with `InProgress`

**Comment by l5io** [comment]
🚀 Preview for commit 6895b07fb72fa7da3d4acf3e44da23bafd019dc5 at: https://62f15009bbb750008af3bfd7--layer5.netlify.app

File: src/assets/data/integrations/IntegrationList.js
Comment (abhiyant-10) [reply src/assets/data/integrations/IntegrationList.js:205]: So , the progresss status is defined by "InProgress" and production by "inProduction" . Problem here is with status of github so can you please confirm is it in progress or production.

Comment (Nikhil-Ladha) [reply src/assets/data/integrations/IntegrationList.js:205]: InProgress

Comment (abhiyant-10) [reply src/assets/data/integrations/IntegrationList.js:205]: Done

**Review by Nikhil-Ladha — APPROVED** [review]

**Comment by l5io** [comment]
🚀 Preview for commit 9bad9dadbed048665d7168da2355075d37553f18 at: https://62f20fa1d3be3f7d68f477c7--layer5.netlify.app

**Pull request merged**
